app/controllers/turns_controller.rb

```diff
@@ -7,7 +9,9 @@ def create
       turn_params[:direction]
     )
 
-    game.turns.create(move: move, player: game.player_num(@user))
+    game.turns.create(move: move, player: @user)
```

Comment:
There is an association between turns and users now right? Then it should be `user: @user`